src/pages/chat/ChattingPage.jsx

```diff
@@ -0,0 +1,172 @@
+      name: 'Amy',
+      context: 'nec non. lorem. luctus ac Donec non, efficitur. diam vitae ame ...',
+      time: '09:25',
+    },
+    {
+      id: '7',
+      profile: require('@assets/images/test_img/test_connectProfile.jpeg'),
+      name: 'Amy',
+      context: 'nec non. lorem. luctus ac Donec non, efficitur. diam vitae ame ...',
+      time: '09:25',
+    },
+  ]
+
+  const [searchTerm, setSearchTerm] = useState('');
+  const [searchData, setSearchData] = useState([]);
+  const [isSearching, setIsSearching] = useState(false);
+
+  const [chatListCount, setChatListCount] = useState(1);
+
+  const handleSearch = () => {
+    if (searchTerm.trim() !== '') {
+      axios.get(`${searchTerm}`)
+        .then(response => {
+          setSearchData(response.data);
+        })
+        .catch(error => {
+          console.error('Error:', error);
+        });
+    }
+  };
+
+  const handleFocus = () => {
+    setIsSearching(true);
+  };
+
+  const handleBlur = () => {
+    setIsSearching(false);
+  };
+
+  const handleCancel = () => {
+    setSearchTerm('');
+    setIsSearching(false);
+    Keyboard.dismiss();
+  };
+
+  return (
+    <View style={ChattingStyles.container}>
+        <ConnectTop style={ChattingStyles.connectTop}/>
+        <TouchableOpacity style={ChattingStyles.iconChatPlus}>
+          <IconChatPlus />
+        </TouchableOpacity>
+        <SafeAreaView style={ChattingStyles.safeAreaView}>
+          <View style={ChattingStyles.containerTextIcon}>
+            <Text style={ChattingStyles.textChattingTitle}>Chatting</Text>
+            <IconBookmark
+              style={ChattingStyles.iconBookmark}
+              onPress={() => navigation.navigate('BookmarkPage')} />
+          </View>
+          <View style={ChattingStyles.containerSearch}>
+            <View style={ChattingStyles.containerSearchIcon}>
+              <TextInput
+                  style={ChattingStyles.search}
+                  placeholder="검색"
+                  value={searchTerm}
+                  onChangeText={setSearchTerm}
+                  onFocus={handleFocus}
+                  onBlur={handleBlur}
+                />
+                {isSearching ? (
+                  <ConnectSearchCancel style={ChattingStyles.searchIcon} onPress={handleCancel} />
+                ) : (
+                  <ConnectSearchIcon style={ChattingStyles.searchIcon} onPress={handleSearch} />
+              )}
+            </View>
+          </View>
+
+          <View style={ChattingStyles.containerMiddle}>
+            <View style={ChattingStyles.containerTab}>
+              <Text style={ChattingStyles.textTab}>1 : 1</Text>
+              <Text style={ChattingStyles.textTab}>그룹</Text>
+            </View>
+            <View style={ChattingStyles.containerReset}>
+              <Text style={ChattingStyles.textReset}>Reset</Text>
+              <ConnectReset />
+            </View>
+          </View>
+
+          {chatListCount != 0 ? (
```

Comment:
```suggestion
{ chatListCount ? (
```
`!= 0`은 생략해도 되겠네요! :)